hypothesis/frontend-shared
# Simplify dev server

This PR simplifies the script that runs the development express server. The changes are:

* Remove mustache entirely. The only template we use is actually static html, so we don't need a templating system.
* Expose the `images` folder as a static source, so that we can access source icon images.

Those two changes make the dev server closer to how the patterns docker container behaves.

With these changes, the `templates` folder might not make a lot of sense anymore. Maybe we can later consider having a `public` folder containing the index.html file and the `images` folder on it. So basically everything which is already static, does not need any building to be served, and we want to be accessible via HTTP.

```tree
public
├── images
│   └── icons
│       └── ...
└── index.html
```

### Testing steps

1. Check out this branch
2. Run dev server: `make dev`
3. Make sure the overall navigation works in http://localhost:4001
4. Verify static images can also be accessed locally http://localhost:4001/images/icons/annotate.svg (any ther icon should also work).
